otx/api/entities/media.py

```diff
@@ -54,4 +54,4 @@ def width(self) -> int:
     @property
     def path(self) -> Optional[str]:
         """Returns the path of the 2D Media object."""
-        raise NotImplementedError
```

Comment:
It's an abstract class, so every method should be overloaded in child classes. Why do we need changing this?

Reply:
No Geti SDK or SC SDK tickets.
Actually, `path` method had been added for OTX-only use-case while OTX1.0 development. SC SDK does not implement it so that abstract method resulted in error. Here again, some module called `path` but it was actually sc_sdk.entities.Image2D. Removing `path` might need some time.
@sungmanc idea?

Reply:
@goodsong81 @sovrasov 

To assure consistency with SC SDK, we could remove all `path` variable inside of `OTX`.
Maybe this could affect the `Action` tasks. I'll check it later and let you know